mono/metadata/object.c

```diff
@@ -7061,17 +7061,17 @@ mono_string_to_utf8 (MonoString *s)
  * \p error to determine whether the conversion was successful.
  * The resulting buffer should be freed with \c mono_free().
  */
-char *
+MONO_API char *
```

Comment:
If the declaration in the `.h` file is marked as `MONO_API` then there is no need to do it in the `.c` file too.

Reply:
I like it but I agree it is debatable.